charts/enterprise-logs/values.yaml

```diff
@@ -97,6 +97,8 @@ config:
     chunk_block_size: 262144
     chunk_encoding: snappy
     chunk_retain_period: 1m
+    wal:
+      dir: /var/loki/wal
```

Comment:
Is it a good idea to put the WAL on ephemeral storage?

Reply:
Also, I think this should be fixed in the loki-distributed chart, if it isn't already.

Reply:
> Is it a good idea to put the WAL on ephemeral storage?

it will be PVC if persistence is enabled.
```yaml
 {{- if not .Values.ingester.persistence.enabled }}
        - name: data
          emptyDir: {}
  {{- else }}
  volumeClaimTemplates:
    - metadata:
        name: data
      spec:
        accessModes:
          - ReadWriteOnce
        {{- with .Values.ingester.persistence.storageClass }}
        storageClassName: {{ if (eq "-" .) }}""{{ else }}{{ . }}{{ end }}
        {{- end }}
        resources:
          requests:
            storage: {{ .Values.ingester.persistence.size | quote }}
  {{- end }}
```

Reply:
:see_no_evil: I've only looked at my local deployment where persistence was not enabled. Should we do that by default, maybe?

Reply:
updated version of loki-distributed . the latest version already contains this fix